packages/functional-tests/tests/react-conversion/oauthPromptNone.spec.ts

```diff
@@ -220,6 +220,36 @@ test.describe('severity-1 #smoke', () => {
       ).toBeVisible();
     });
   });
+
+  test('redirects if return_on_error=true and not signed in', async ({
```

Comment:
Just a drive by (will do some testing a bit later this afternoon), but does this also work if the param is entirely omitted? IIRC on backbone the default is to consider `return_on_error` to be true, and `false` is mainly intended for automated tests

Reply:
Oh good point.   Yeah, it also works when it's the default behavior.